resources/carrier/en/382.txt

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2016 The Libphonenumber Authors
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+38260|m:tel
+38263|Telenor
+38266|Telekom
```

Comment:
The reason I ask is that we have Crnogorski Telekom in the ITU doc (https://www.itu.int/dms_pub/itu-t/oth/02/02/T02020000DA0002PDFE.pdf).

The other thing is we have Mtel in the ITU doc, not m:tel.